typescript/sdk/src/router.ts

```diff
@@ -34,10 +42,9 @@ export type ConnectionClientConfig = {
   mailbox: types.Address;
   interchainGasPaymaster: types.Address;
   interchainSecurityModule?: types.Address;
+  owner?: types.Address;
```

Comment:
is this intentional? not sure I see this in use anywhere

Reply:
Hmm yeah must have missed this when I was cleaning up